SummerProject/static/src/components/Sidebar/style.css

```diff
@@ -0,0 +1,186 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans&display=swap');
```

Comment:
Do we really need a third-party dependency for fonts? I think no and it looks like you can get rid of this `import`

Reply:
You are right, I removed it